### PR TITLE
Fixes #7158: Modify ncf api to adapt to new class_prefix

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7158

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
Great! merging!

File: tools/ncf.py [hunk 1]
```diff
@@ -455,7 +458,11 @@ def generate_technique_content(technique_metadata):
       arg_value = ""
     class_context = canonify_class_context(method_call['class_context'])
 
-    content.append('    "method_call" usebundle => '+method_call['method_name']+'('+arg_value+'),')
+    if 'promiser' in +method_call:
```
Comment (VinceMacBuche) [inline tools/ncf.py]: Can you remove the + before method_call ? ;)

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**